homeassistant/components/mqtt/climate.py

```diff
@@ -99,6 +103,12 @@
 CONF_HOLD_STATE_TOPIC = "hold_state_topic"
 CONF_HOLD_LIST = "hold_modes"
 
+CONF_HUMIDITY_COMMAND_TEMPLATE = "target_humidity_command_template"
+CONF_HUMIDITY_COMMAND_TOPIC = "target_humidity_command_topic"
+CONF_HUMIDITY_STATE_TEMPLATE = "target_humidity_state_template"
+CONF_HUMIDITY_STATE_TOPIC = "target_humidity_state_topic"
```

Comment:
May be we can leave the `target_` prefix away. The topics get much the same like the temperature topics. I know I asked you at first to add the prefix.

Reply:
That's what I tried to do at the beginning, but when I went to add them to `abbreviations.py`, I found out that they already exist with the `target_` prefix (by the MQTT Humidifier): https://github.com/home-assistant/core/blob/c56806a676e8c2482c6ee9e5880c2d2f01af59ef/homeassistant/components/mqtt/abbreviations.py#L88-L91

We could drop the `target_` prefix, but then we'd have to come up with a new abbreviation.

Reply:
Then lets leave it the way it is!